CounterApp/script.js

```diff
@@ -0,0 +1,14 @@
+
+        const buttonMinus= document.getElementById("minus-button")
+        const buttonPlus= document.getElementById("plus-button")
+        const span= document.getElementById("value")
+        
+        let value= 0;
```

Comment:
bora dar um espaçamento nessas atribuições?
```suggestion
        const buttonMinus = document.getElementById("minus-button")
        const buttonPlus = document.getElementById("plus-button")
        const span = document.getElementById("value")
        
        let value = 0;
```